src/landing-page/types.ts

```diff
@@ -0,0 +1,28 @@
+import { ViewerChannelSettings } from "../aics-image-viewer/shared/utils/viewerChannelSettings";
+
+// TBD what URL parameters to include here
+export type ViewerArgs = {
+  baseurl: string;
+  cellid: number;
+  cellPath: string;
+  fovPath: string;
+  fovDownloadHref: string;
+  cellDownloadHref: string;
+  viewerSettings: ViewerChannelSettings;
+};
+
+export type DatasetEntry = {
+  name: string;
+  description: string;
+  loadParams: ViewerArgs;
+};
+
+export type ProjectEntry = {
+  name: string;
+  description: string;
+  publicationLink?: URL;
+  publicationName?: string;
+  loadParams?: ViewerArgs;
+  datasets?: DatasetEntry[];
```

Comment:
Clarifying question: it's possible for a project to both have loadable data itself and have a list of datasets with loadable data? (I think this is fine if so, not implying that this type needs changing)

Reply:
Yes, correct!